apiary.apib

```diff
@@ -327,6 +327,27 @@ _This will only affect the version for those API calls and won't affect any othe
 
 We list all backwards-incompatible changes here. As described above, new additions and forwards-compatible changes don’t need a new API version and will not appear in this list.
 
+#### Beta
+
+- Previously, the `invoices.info` endpoint returned an _non-existing_ or _empty_ `extended_description` as an empty string, while `invoices.draft` and `invoices.update` didn't allow an empty string as `extended_description`.  
+  From now on, `invoices.info` will return an _non-existing_ or _empty_ `extended_description` as `null`.
+
+- Invoices can now have three statuses (`draft`, `outstanding` or `matched`) instead of two (`draft` or `booked`).  
+  Therefore the `status` property in `invoices.list` and `invoices.info` will either return `outstanding` or `matched` for invoices that were previously just `booked`.  
+  This is also reflected in the `status` filter on `invoices.list`: to allow you to still query all `outstanding` and `matched` invoices (previously all `booked`) , it now *only* accepts an array of filter values (eg. `["outstanding", "matched"]` or `["draft"]`).
+
+- The property `local_business_number` was renamed to `national_identification_number` on the following endpoints:  
+  `companies.info`,  
+  `companies.list`,  
+  `companies.add`,  
+  `companies.update`,  
+  `invoices.info`  
+  and `creditNotes.info`.
+
+- Added the bookyear name to the `invoice_number` on `invoices.list` and `invoices.info`.
```

Comment:
I order to be consistent with language, we should use passive voice here ("bookyear name was added") or use active voice in the previous bullet point "Renamed x to x in the following endpoints".

Reply:
better in line now?